apps/data-types.md

```diff
@@ -640,3 +640,22 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | productNumber | string | The product number. |
 | watchers | number | The number of watchers. |
 | links | array of [link](page:apps-data-types#link) | The link to the watched product. |
+
+## webhook
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| id | string | The unique identifier of the webhook. |
+| active | boolean | The status of the webhook. Can be `true` or `false`. |
+| eventType | string | The event that triggers the webhook. |
```

Comment:
The event that triggers the webhook. Can be ORDER_CREATE, ORDER_UPDATE, ORDER_DELETE or APP_UNINSTALL